conferences/abstracts/OSF19_Demo_submission.md

```diff
@@ -0,0 +1,91 @@
+
+
+## Demo presentation content
+
+*Authors are asked to submit a short proposal that describes the main contributions of the demo. Proposals should contain a brief abstract, place an emphasis on the motivation for the work, and summarize contributions being presented.* 
+
+*	_Proposal length should be a minimum of 300 words and should not exceed 500 words._
+*	_The language of the conference will be English; therefore, the abstract must be in this language._
+*	_All submitted abstracts will be peer-reviewed by members of the Conference Programme Committee based on the criteria mentioned on the event website._
+
+*Authors of accepted demo proposals will be provided instructions for the demo presentations. Stands with a screen will be provided to display all accepted demos.*
+
+Reproducible research is necessary to ensure that scientific work can be trusted. Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*The Turing Way* is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". Our goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end. So far, the book (https://the-turing-way.netlify.com) includes chapters and training material on reproducibility, open research, version control, collaborating on GitHub/GitLab, research data management, reproducible environments, analysis testing, reviewing, continuous integration, reproducible research with Make and risk assessment. Further chapters currently in progress include coding style for reproducibility, credit for reproducible research and reproducible data analysis pipelines for machine learning, and will build upon Alan Turing Institute case studies. 
+
+The project also demonstrates open and transparent project management and communication with future users, as it is openly developed at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way. Any and all questions, comments and recommendations are welcome. All resources for the "Boost your research reproducibility with Binder" and "Build your own BinderHub" workshops we have delivered, as well as how to organise a Book Dash (a one day book sprint), are also open and will soon be submitted to the Journal of Open Source Education (JOSE).
```

Comment:
This is a great paragraph. And basically the thing I was saying to emphasise in my comment above 😆